cli.js

```diff
@@ -41,13 +41,13 @@ var cli = meow([
 	'  ava [<file|folder|glob> ...]',
 	'',
 	'Options',
-	'  --init       Add AVA to your project',
-	'  --fail-fast  Stop after first test failure',
-	'  --serial     Run tests serially',
-	'  --require    Module to preload (Can be repeated)',
-	'  --tap        Generate TAP output',
-	'  --verbose    Enable verbose output',
-	'  --no-cache   Disable the transpiler cache',
+	'  --init           Add AVA to your project',
+	'  --fail-fast      Stop after first test failure',
+	'  -s, --serial     Run tests serially',
```

Comment:
The standard is short flag first, but it does not look very good to me. Of course this thing is way too small to argue over, so if guys prefer the other way, fine by me ;)